examples/django_chat/chat/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import (
+    url, patterns, include
+)
+
+import sdjango
+
+from .views import socket_base
+
+
+urlpatterns = [
+    url(r'^socket\.io', include(sdjango.urls)),
```

Comment:
This defeats the purpose of using a middleware to do the routing between the socket and django endpoints. Note that in the middleware the `/socket.io` endpoint can be configured to be anything you want, while here it is hardcoded, so the ability to rename the endpoint is broken. I think I prefer a soluiton that does not require hacks around the middleware. I will think about how to get the request object using a cleaner way.